csm/conf/csm.post_upgrade.tmpl.1-node

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Please refer - https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/313426367/Mini-Provisioner+Specifications
I do not think templates are required for upgrade

Reply:
@pawankumarsrivastava As per the mail by andrey, component can add the args if they waant. So current implimentation

> From: Andrey Kononykhin <andrey.kononykhin@seagate.com>
Sent: 28 May 2021 19:41
To: Nilesh Govande <nilesh.govande@seagate.com>; Nitin Nimran <nitin.nimran@seagate.com>; Anurag Modi <anurag.modi@seagate.com>
Cc: Vimal Raghwani <vimal.raghwani@seagate.com>; Yash Pise <yashodhan.pise@seagate.com>; Jaikumar Gidwani <jaikumar.gidwani@seagate.com>; Ujjwal Lanjewar <ujjwal.lanjewar@seagate.com>
Subject: Re: Fixing interface semantics
 >
>
> 
> These are just examples. And I don't think for these commands we need that.
> 
> In case of some component's own script (command) which require that config (confstore url), sure, it should be there.
> 
> I guess in current implementation args is mandatory if cmd is present. I tend to make it optional in any case.
> 
> Please check EOS-20788 Support for upgrade related mini API spec which was created to address the necessary changes from Provisioner side.
> 
> Thanks

Reply:
Also the https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/313426367/Mini-Provisioner+Specifications states some rules, where its mentioned that args can be optional.
![image](https://user-images.githubusercontent.com/66424882/120279770-795e2380-c2d4-11eb-961e-32896701c53e.png)